src/config/dynamic-theme-fixes.config

```
@@ -10409,7 +10409,14 @@ div.fly-but-wrap.left.relative > span
 
 reddit.com
 
+INVERT
+[role="slider"]
+[style^="height"]
```

Nope, this will need to be a more strict, I know they are applying to posts that are out of the content view these `height` so they appear out of the content view, I don't like to apply any invert over those.

```suggestion
video ~ div [style^="height"]
```

---

Ah nice use of the `~` selector. That's a handy way to make it more targeted, thanks.